src/pages/workshop/speakers.md

```diff
@@ -27,4 +27,4 @@ template: page
 
 ## NLLP 2023 Keynote: Jerrold Soh
 
-[<img src="/images/NLLP2023-4.png" width="600"/>](https://scholar.google.com/citations?user=Q_BoKqYAAAAJ&hl=en)
+[<img src="static/images/NLLP 2023-4.png" width="600"/>](https://scholar.google.com/citations?user=Q_BoKqYAAAAJ&hl=en)
```

Comment:
No need to use `static/` path, the content of static gets dumped into the root of the website 

```suggestion
[<img src="/images/NLLP 2023-4.png" width="600"/>](https://scholar.google.com/citations?user=Q_BoKqYAAAAJ&hl=en)
```